lib/portlayer/exec/commit.go

```diff
@@ -92,6 +92,12 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 		// clear the spec as we've acted on it - this prevents a reconfigure from occurring in follow-on
 		// processing
 		h.Spec = nil
+
+		// reconfigure vm display name to containerName-containerID
+		if err = c.UpdateDisplayName(ctx); err != nil {
+			return err
```

Comment:
Can we do the rename by setting it in the config spec (`h.Spec`)? This really should not be a separate step here, but should be done as part of the reconfigure.